frontend/typed_args.py

```diff
@@ -74,52 +75,120 @@ def RestWords(self):
 
     ### Typed positional args
 
+    def BlamePos(self, pos=-1):
```

Comment:
I would call this BlameLoc() since it returns loc_t, I think we're relatively consistent about that

Reply:
I originally wanted one `BlameLoc()`, but it felt awkward to have two kwargs for the positional and named arg cases. So, there are separate functions: `BlamePos()` and `BlameNamed()`